### PR TITLE
Release 0.2.4 blog

### PR DESCRIPTION
**Pull request by betarelease**

### CODE REVIEW

**Comment by betarelease** [comment]
@karolh2000 this is the release announcement for 0.2.4. I would like to publish yours first and add a link to it in this blog.

**Comment by karolh2000** [comment]
> @karolh2000 this is the release announcement for 0.2.4. I would like to publish yours first and add a link to it in this blog.

@betarelease Sure, working on the blog PR entry, should be publish in a few, will add you as a reviewer.

**Comment by karolh2000** [comment]
> > @karolh2000 this is the release announcement for 0.2.4. I would like to publish yours first and add a link to it in this blog.
> 
> @betarelease Sure, working on the blog PR entry, should be publish in a few, will add you as a reviewer.

This is the VSCode Pyrsia extension PR (https://github.com/pyrsia/pyrsia.github.io/pull/174), feel free to add reviewers as needed. Thanks!

**Review by fishseabowl — APPROVED** [review]
lgtm

File: blog/2023-02-14-pyrsia-0.2.4-released.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+members.
+
+We are happy to announce the release of [version 0.2.4](https://github.com/pyrsia/pyrsia/releases/tag/v0.2.4) today!
+
+We are our continued journey on a small & frequent release schedule. This is to allow us to release consistently and in
+small increments as we decided.
+
+A couple of big announcements we would like to make with this release:
+
+* We are happy to announce the release of a VSCode extension for Pyrsia - this will help developers bring secure images from Pyrsia into their development.
```
Comment (dasmanas) [inline blog/2023-02-14-pyrsia-0.2.4-released.md:19]: Should we also provide the Youtube link https://www.youtube.com/watch?v=4ALKmtWmots here?

Comment (dasmanas) [reply blog/2023-02-14-pyrsia-0.2.4-released.md]: resolved wrt https://github.com/pyrsia/pyrsia.github.io/pull/175

**Review by dasmanas — APPROVED** [review]
LG

**Pull request merged**